joplin/templates/wagtailadmin/pages/search_header.html

```diff
@@ -11,22 +11,12 @@ <h1 class="coa-search-results-h1">
     <div>
         <!-- From wagtail/admin/templates/wagtailadmin/shared/menu_search.html -->
         {% if search_url %}
-            <form class="col search-form nav-search" action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}" method="get" novalidate>
-                <ul class="fields">
-                    {% for field in search_form %}
-                        {% include "wagtailadmin/shared/field_as_li.html" with field=field field_classes="field-small iconfield" input_classes="icon-search" %}
-                    {% endfor %}
-                    <li class="submit visuallyhidden"><input type="submit" value="Search" class="button" /></li>
-                </ul>
-            </form>
-        {% endif %}
-        <!-- <form class="nav-search" action="{% url 'wagtailadmin_pages:search' %}" method="get">
-            <div>
-                <label for="menu-search-q">{% trans "Search" %}</label>
-                <input type="text" id="menu-search-q" name="q" placeholder="{% trans 'Search' %}" />
-                <button class="button" type="submit">{% trans "Search" %}</button>
+            <div class="col search-form nav-search">
+                {% for field in search_form %}
+                    {% include "wagtailadmin/shared/field_as_li.html" with field=field field_classes="field-small iconfield" input_classes="icon-search" %}
+                {% endfor %}
```

Comment:
Here is the root of where the issue was... 
- the "action" was redundant, as the imbedded ajax request was already doing the querying for us.
- So removing the actual html form behavior just let the input be an input, and, filtering now doesn't have to care what the input state of that form is.
- Before it freaked out when it saw new values in the search box and did and alert("you have unsaved data")

Reply:
ah, so it was trying to do too much?